pkgs/tools/security/wordlists/nmap.nix

```diff
@@ -0,0 +1,34 @@
+{ fetchFromGitHub
+, lib
+, nmap
+, stdenv
+}:
+
+let
+  shared = "nmap.lst";
+
+in stdenv.mkDerivation {
+  pname = "nmap.lst";
+  version = "unstable-2020-10-19";
```

Comment:
Is this still under the free license or the unfree?

Reply:
Oh wow, I am just learning about the `nmap` license update...

As for the list of passwords, it does not seem to have been updated (prior to the license change) since 2012! So IMHO, we can go with the `free` version.

I updated the commit (and related sha), to specifically pin a commit prior to the license change.